source/other-document-dbs.txt

```diff
@@ -0,0 +1,17 @@
+=====================================
+Other Document Database Compatibility
+=====================================
+
+.. toctree::
+   :titlesonly:
+   :maxdepth: 1
+
+   /documentdb-support
+   /cosmosdb-support
+
+See the following pages for information about other document databases' compatibility
+with MongoDB:
+
+- :ref:`documentdb-compat`
+- :ref:`cosmosdb-compat`
+
```

Comment:
S: not sure if this is intentional, but this landing page isn't currently accessible from the sidebar.  If you did mean to make this page viewable, add to the snooty.toml file.

Reply:
I hadn't intended it to be visible from the sidebar, but I might as well.